packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -111,10 +126,11 @@ export default class RtcMetrics {
         metrics: [
           {
             type: 'webrtc',
-            version: '1.0.1',
+            version: '1.1.0',
```

Comment:
q: what is this?

Reply:
A version of the payload being sent. Not sure if it is used for anything.